### PR TITLE
User authentication

### PR DESCRIPTION
**Pull request by mariomelchor**

### CODE REVIEW

File: client/public/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+body {
+  background: #f1f1f1;
+}
+.login {
+  width: 320px;
+  padding: 8% 0 0;
+  margin: auto;
+}
```
Comment (saloiofun) [inline client/public/css/styles.css]: The style css is called from index.css in the src folder. Please add the styles on that file

File: client/public/index.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link href="%PUBLIC_URL%/css/styles.css" rel="stylesheet">
     <title>EviteHub</title>
```
Comment (saloiofun) [inline client/public/index.html]: Same as above. We don't need to call style for font and custom styles in the HTML

File: client/src/App.js [hunk 3]
```diff
@@ -181,4 +183,4 @@ App.propTypes = {
   theme: PropTypes.object.isRequired,
 }
 
-export default withStyles(styles, { withTheme: true })(App);
\ No newline at end of file
+export default withStyles(styles, { withTheme: true })(App);
```
Comment (saloiofun) [inline client/src/App.js]: Remove semicolon

File: client/src/pages/login/Login.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import API from "../../utils/Api";
+import Card, { CardContent } from 'material-ui/Card';
+import Button from 'material-ui/Button';
+import TextField from 'material-ui/TextField';
+
```
Comment (saloiofun) [inline client/src/pages/login/Login.js]: Use single quotation and remove semicolon

File: client/src/pages/login/Login.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+class Login extends Component {
+  // Setting the initial values
+  state = {
+    username: "",
+    password: "",
+    loggedin: false,
+    user: {}
+  };
```
Comment (saloiofun) [inline client/src/pages/login/Login.js]: remove semicolon

File: client/src/pages/login/Login.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+  // handle any changes to the input fields
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    // Set the state for the appropriate input field
+    this.setState({
+      [name]: value
+    });
+  };
```
Comment (saloiofun) [inline client/src/pages/login/Login.js]: remove semicolon

File: client/src/pages/login/Login.js [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+  // When the form is submitted
+  handleFormSubmit = event => {
+    event.preventDefault();
```
Comment (saloiofun) [inline client/src/pages/login/Login.js]: remove semicolon

File: client/src/pages/login/Login.js [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+          { this.state.user.email }<br />
+          { this.state.user.password }<br />
+          { this.state.user.firstName }<br />
+          { this.state.user.lastName }<br />
+      </div>
+    );
+  }
+}
+
+export default Login;
```
Comment (saloiofun) [inline client/src/pages/login/Login.js]: remove semicolon

**Review by saloiofun — CHANGES_REQUESTED** [review]
Please verify

**Comment by mariomelchor** [comment]
Ok I think in got everything. Please review. Thx

**Review by saloiofun — APPROVED** [review]

**Pull request merged**